From bcf8575844c816fab2f7d736620637b3575b7932 Mon Sep 17 00:00:00 2001
From: Cedric Poignet <cedric.poignet@intel.com>
Date: Mon, 10 Jun 2013 19:53:21 +0200
Subject: [PORT FROM MAIN][BT] Remove bluedroid if BOARD_HAVE_BLUETOOTH is false

BZ: 110380

It removes useless Bluedroid HAL libraries in case the
Bluetooth is not supported by the platform.

Change-Id: I56d12ed9e96e957e5a340a5acfab14dcecc60b9b
Orig-Change-Id: I53a2ce2597a49aa60b74c058d9fde23938e19a56
Category: device enablement
Domain: CWS.BT-Common
Origin: internal
Upstream-Candidate: yes
Signed-off-by: Cedric Poignet <cedric.poignet@intel.com>
Reviewed-on: http://android.intel.com:8080/122990
Reviewed-by: Gros, Julien <julien.gros@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
---
 Android.mk |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/Android.mk b/Android.mk
index 5f2d274..e8ea576 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,3 +1,4 @@
+ifneq ($(BOARD_HAVE_BLUETOOTH),false)
 LOCAL_PATH := $(call my-dir)
 
 # Setup bdroid local make variables for handling configuration
@@ -14,3 +15,4 @@ include $(call all-subdir-makefiles)
 # Cleanup our locals
 bdroid_C_INCLUDES :=
 bdroid_CFLAGS :=
+endif
-- 
1.7.4.1

